### DISCUSSION TITLE
32-bit Installer Should Install to "Progam Files (x86)" on 64-bit Machines

### DISCUSSION THREAD

**Discussion by AntumDeluge** [d2phap/ImageGlass#289] [General] [answered]
I don't know that this is considered a "bug".

The installer for version 4.5.11.27 installs a 32-bit version of ImageGlass & defaults to *C:\Program Files*. This is correct for 32-bit systems. But on 64-bit Windows systems, the folder *C:\Program Files (x86)* is reserved for 32-bit software. So the installer should default to this.

**Comment by d2phap** [comment, marked as answer]
Hi @AntumDeluge 
ImageGlass is built with _AnyCPU_ config, it will run both 32-bit and 64-bit system.
In the 64-bit system, it will be installed to _C:\Program Files_
